woocommerce/woocommerce-ios
# Product selector: Revert changes to the product selector in linked products flow

⚠️ Please make sure that #6696 & #6715 are reviewed and merged first ⚠️
Closes: #6489


### Description

Since the new product selector has been completed, this PR reverts changes to the legacy product selector in the linked products flow. We'll need to migrate this flow to use the new product selector, but it's out of the scope of #6489 so we'll handle this later in a separate task.

### Testing instructions

- Navigate to the Products tab > select a Product > Add more details > Linked products > Add Products > Add Products.
- Notice that the product selector in this flow works exactly like before.

### Screenshots

<img src="https://user-images.githubusercontent.com/5533851/165265601-6b98e33d-74e0-4874-8d26-2b9a75ecdf16.png" width=320 />

---
